kubernetes/structures.go

```diff
@@ -613,3 +613,19 @@ func expandNodeSelectorTerms(l []interface{}) []api.NodeSelectorTerm {
 	}
 	return obj
 }
+
+func flattenPersistentVolumeMountOptions(in []string) *schema.Set {
+	var out = make([]interface{}, len(in), len(in))
+	for i, v := range in {
+		out[i] = string(v)
+	}
+	return schema.NewSet(schema.HashString, out)
+}
+
+func expandPersistentVolumeMountOptions(s []interface{}) []string {
```

Comment:
Why not use `schemaSetToStringArray` and make this generic?

Reply:
Ignorance of that func's existence?  I will update the PR.